src/containers/pages/review_form.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from 'react';
+import {Bootstrap, Grid, Row, Col, Container, ProgressBar} from 'react-bootstrap';
+
+class RatingBar extends React.Component{
+    render(){
+        return(
+            <div>
+                {this.props.value}
+            </div>
+        );
+    }
+}
+class Date extends React.Component{
+    render(){
+        return(
+            <div>
+                Date: {this.props.value}  
+            </div>
+        );
+    }
+}
+class Semester extends React.Component{
+    render(){
+        return(
+            <div>
+                Semester: {this.props.value}
+            </div>
+        );
+    }
+}
+class Text extends React.Component{
+    render(){
+        return(
+            <div>
+                {this.props.value}
+            </div>
+        );
+    }
+}
+class Professor extends React.Component{
+    render(){
+        return(
+            <div>
+                {this.props.value}      
+            </div>
+        );
+    }
+}
+export default class ReviewForm extends React.Component {
+    renderRatingsBar(rating){
```

Comment:
It would be simpler to just inline the ProgressBar component

Reply:
what do you mean by inline?

Reply:
This function only returns some React elements, it would be simpler to just put these elements directly into your render() function